### PR TITLE
Change images to yolks to prevent confusion

### PR DESCRIPTION
**Pull request by pascal-zarrad**
### Documentation adjustment
There was some confusion in the Discord about where to contribute new images to as yolks is the new images repository.

This PR solves future headache by:

- Changing `my image repo` to `my yolks repo` to tell people the new repository for images is yolks.
- Changing the repository url to `parkervcp/yolks`
  - Tells contributors to use the yolks repo
  - Tells contributors also that they should contribute to `parkervcp/yolks` instead of the main `pterodactyl/yolks` repository

### All Submissions:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### CODE REVIEW

**Comment by pascal-zarrad** [comment]
Maybe some hint that the parkervcp/images repo should be checked for an existing image would be good too.

What is your opinion about that?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
     Only use what is absolutely needed.  
 
 3. Try to stay in the stock containers.  
-    If you need something in a container PR it to [my image repo](https://github.com/parkervcp/images) where I can review and pull up to the main repo.
+    If you need something in a container PR it to [my yolks repo](https://github.com/parkervcp/yolks) where I can review and pull up to the main repo.
```
Comment (Software-Noob) [inline CONTRIBUTING.md:12]:
```suggestion
    If you need something in a container, open a PR in [my yolks repo](https://github.com/parkervcp/yolks) where I can review and pull it up to the main repo. In addition, there is a larger quantity of [images here](https://github.com/parkervcp/images) for you to use that have not been migrated to Yolks yet.
```

Something along these lines would provide options for both

Comment (pascal-zarrad) [reply CONTRIBUTING.md]: The suggestion has been commited

**Review by Software-Noob — APPROVED** [review]
LGTM. I'll leave this up to @parkervcp to decide

**Comment by parkervcp** [comment]
I would say update it but keep a reference to the old images repo until it's completely gone.

**Pull request merged**